camel/toolkits/search_toolkit.py

```diff
@@ -322,20 +322,29 @@ def _parse_wolfram_result(self, result) -> Dict[str, Any]:
 
         # Loop through each pod to extract the details
         for pod in result.get('pod', []):
+            # Handle the case where subpod might be a list
+            subpod_data = pod.get('subpod', {})
+            if isinstance(subpod_data, list):
+                # If it's a list, get the first item for 'plaintext' and 'img'
+                description = subpod_data[0].get('plaintext', '')
+                image_url = subpod_data[0].get('img', {}).get('@src', '')
```

Comment:
Maybe do something like 
description, image_url = next(((data['plaintext'], data['img']) for data in subpod_data if "plaintext" in data and "img" in data), ("",""))
I just don't really like hard code the index.